src/nccl_ofi_net.c

```diff
@@ -46,7 +46,7 @@ int nic_dup_conns = 0;
    This variable will be updated during init (hence, can not be
    const), but will not change during execution.  Therefore, it may be
    read in the polling loop without protection of a lock. */
-size_t cq_read_count = 1;
+size_t cq_read_count = 4;
```

Comment:
What we talked about on the call was just making this number bigger everywhere.  This seems overly complicated for no reason.

Reply:
Ok. Let's close this PR completely and let me post one that just increases default to 16.